src/plugins/kibana_usage_collection/server/plugin.ts

```diff
@@ -126,7 +125,6 @@ export class KibanaUsageCollectionPlugin implements Plugin {
     const getCoreUsageDataService = () => this.coreUsageData!;
 
     registerUiCounterSavedObjectType(coreSetup.savedObjects);
-    registerUiCountersRollups(this.logger.get('ui-counters'), pluginStop$, getSavedObjectsClient);
     registerUiCountersUsageCollector(usageCollection, pluginStop$);
```

Comment:
As we are not rolling them up any longer, we should stop registering the SavedObject type.

Also, we should not perform the `soClient.find` request to fetch them: 
https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/kibana_usage_collection/server/collectors/ui_counters/register_ui_counters_collector.ts#L106-L113

What do you think?

Reply:
> we should stop registering the SavedObject type

If it's no longer used, for sure. Note that you will need to add it here:

https://github.com/elastic/kibana/blob/ba0833f42b89a0b62bccd1d8c04bf9c799f32258/src/core/server/saved_objects/migrations/core/unused_types.ts#L14-L16